packages/web-components/fast-foundation/src/text-field/text-field.ts

```diff
@@ -118,12 +110,18 @@ export class FASTTextField extends FormAssociatedTextField {
      * HTMLAttribute: maxlength
      */
     @attr({ converter: nullableNumberConverter })
-    public maxlength: number;
-    protected maxlengthChanged(): void {
-        if (this.proxy instanceof HTMLInputElement) {
-            this.proxy.maxLength = this.maxlength;
-            this.validate();
+    public maxlength: number | null;
+    protected maxlengthChanged(
+        prev: number | null | undefined,
+        next: number | null
+    ): void {
+        if (typeof next === "number") {
+            this.proxy?.setAttribute("maxlength", next.toString());
```

Comment:
Do we need to set that attribute to a string given the `nullableNumberConverter`?

Reply:
Nevermind - I think that's not the case because we're setting on the proxy here.